jenkins/DataLoader/OpensearchBackup.jenkinsfile

```diff
@@ -0,0 +1,85 @@
+import groovy.json.JsonOutput
```

Comment:
the library import should be added at the top so common functions like notifications can be added